lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Form.js

```diff
@@ -231,6 +231,9 @@ qx.Class.define("callbackery.ui.plugin.Form", {
                             if (item.set && item.set.readOnly && data){
                                 statusData[item.key] = data[item.key];
                             }
+                            if (item.widget == 'selectBox' || item.widget == 'comboBox') {
+                                that._form.setSelectBoxSelection(item.key, data[item.key]);
+                            }
```

Comment:
```suggestion
                            if (item.set && item.set.readOnly && data){
                                if (item.widget == 'selectBox' || item.widget == 'comboBox') {
                                    that._form.setSelectBoxSelection(item.key, data[item.key]);
                                } else {
                                    statusData[item.key] = data[item.key];
                                }
                            }
```

I would suggest keeping the limitation of only changing readOnly widgets